### ISSUE TITLE
Uncaught (in promise) DOMException: The play() request was interrupted by a call to pause()

### ISSUE THREAD

**Issue by alexlii1971** [dkaoster/scrolly-video#55]
Hello,

I encountered the error, and please check the screenshot:

![Lightshot screenshot (1)](https://github.com/dkaoster/scrolly-video/assets/15896194/17c69c6d-6855-4717-9b53-bcb4d389383c)

Any suggest to fix it please?

Thanks

**Comment by cvalues** [comment]
same here.

**Comment by wkrawczak** [comment]
Can confirm on Chrome 113.0.5672.126

**Comment by pixelastudio** [comment]
same problem

**Comment by Paolino95** [comment]
Hi guys, maybe I have a suggest about this issue: 

In `ScrollyVideo.js`, in `setTargetTimePercent` function, line 416-420 or above, there is this if: 

```
// Play the video if we are in video mode
if (!this.canvas) this.video.play();
```

this is the play() request that is interrupted by the call to pause, but on this call, no consideration is given as to whether the video should actually start or not. 
Having previously calculated the targetTime in this function, 
I made this modification: 

` if (!this.canvas && this.targetTime > 0) this.video.play();`

And I no longer have that error. 
However, I don't know if this brings real problems in video-only mode situations, because I haven't had the opportunity and the environment to test it. 

In all others scrollyVideo contexts I think it may be a suitable solution. 

Let me know and have a good day

**Comment by dkaoster** [comment]
Just released https://github.com/dkaoster/scrolly-video/releases/tag/0.0.16, which should fix this issue! Let me know if that solves the problem, otherwise I can reopen this issue and continue testing.

**Comment by tommyiaco** [comment]
We encountered the same issue, and it was resolved with version `0.0.16`. Appreciate the assistance!
As a point of context, we are utilizing the Vue component.
